src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -684,12 +778,14 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             ChatId chatId,
             bool disableNotification = default,
             int replyToMessageId = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            bool allowSendingWithoutReply = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.